x-pack/filebeat/module/o365/audit/config/pipeline.js

```diff
@@ -0,0 +1,786 @@
+    builder.Add("saveFields", new processor.Convert({
+        fields: [
+            {from: 'o365audit.ObjectId', to: 'url.original'},
+            {from: 'o365audit.SourceRelativeUrl', to: 'file.directory'},
+            {from: 'o365audit.SourceFileName', to: 'file.name'},
+            {from: 'o365audit.SourceFileExtension', to: 'file.extension'},
+        ],
+        ignore_missing: true,
+        fail_on_error: false
+    }));
+    builder.Add("setEventCategory", new processor.AddFields({
+        target: 'event',
+        fields: {
+            category: 'file',
+        },
+    }));
+    builder.Add("mapEventType", makeMapper({
+        from: 'o365audit.Operation',
+        to: 'event.type',
+        mappings: {
+            'FileAccessed': 'access',
+            'FileDeleted': 'deletion',
+            'FileDownloaded': 'access',
+            'FileModified': 'change',
+            'FileMoved': 'change',
+            'FileRenamed': 'change',
+            'FileRestored': 'change',
+            'FileUploaded': 'creation',
+            'FolderCopied': 'creation',
+            'FolderCreated': 'creation',
+            'FolderDeleted': 'deletion',
+            'FolderModified': 'change',
+            'FolderMoved': 'change',
+            'FolderRenamed': 'change',
+            'FolderRestored': 'change',
+        },
+    }));
+    return builder.Build();
+}
+
+function exchangeMailboxSchema(debug) {
+    var builder = new PipelineBuilder("o365.audit.SharePointFileOperation", debug);
+    builder.Add("saveFields", new processor.Convert({
+        fields: [
+            {from: 'o365audit.MailboxOwnerUPN', to: 'user.email'},
+            {from: 'o365audit.LogonUserSid', to: 'user.id'},
+            {from: 'o365audit.LogonUserDisplayName', to: 'user.full_name'},
+            {from: 'o365audit.OrganizationName', to: 'organization.name'},
+            {from: 'o365audit.OriginatingServer', to: 'server.address'},
+            {from: 'o365audit.ClientIPAddress', to: 'client.address'},
+            {from: 'o365audit.ClientProcessName', to: 'process.name'},
+        ],
+        ignore_missing: true,
+        fail_on_error: false
+    }));
+    return builder.Build();
+}
+
+function dataLossPreventionSchema(debug) {
+    var builder = new PipelineBuilder("o365.audit.DLP", debug);
+    builder.Add("setEventFields", new processor.AddFields({
+        target: 'event',
+        fields: {
+            kind: 'alert',
+            category: 'file',
+            type: 'access',
+        },
+    }));
+
+    builder.Add("saveFields", new processor.Convert({
+        fields: [
+            // SharePoint metadata
+            {from: 'o365audit.SharePointMetaData.From', to: 'user.id'},
+            {from: 'o365audit.SharePointMetaData.FileName', to: 'file.name'},
+            {from: 'o365audit.SharePointMetaData.FilePathUrl', to: 'url.original'},
+            {from: 'o365audit.SharePointMetaData.UniqueId', to: 'file.inode'},
```

Comment:
It's documented as `UniqueId` but the API is producing `UniqueID`. Was just being a little too defensive in here, just in case they realize the discrepancy and change the generated field instead of the docs.